client/source/components/questionsAnswers/QuestionForm.jsx

```diff
@@ -0,0 +1,66 @@
+import React, { useCallback, useState } from 'react';
+import { Form, InputSubmit } from '../shared/form/FormStyles';
+import FormInput from '../shared/form/FormInput';
+import FormTextarea from '../shared/form/FormTextarea';
+
+const initialFormInput = {
+  question: '',
+  name: '',
+  email: ''
+};
+
+const QuestionForm = function CreateQuestionFormComponent({ handlePost }) {
+  const [formInput, setFormInput] = useState(initialFormInput);
+
+  const clearForm = () => {
+    setFormInput(initialFormInput);
+  };
+
+  const handleFormInputChange = useCallback((event) => {
+    setFormInput({
+      ...formInput,
+      [event.target.name]: event.target.value
+    });
+  }, [formInput]);
+
+  return (
```

Comment:
Super clean - I'll likely align to what you have here :)

Reply:
🙌